### PR TITLE
Networks

### PR DESCRIPTION
**Pull request by kevinylee1991**
Currently only takes a name property (can add support for cidr, etc later).

One question I have is where should helper functions be located? Ex. cidr validations, IP address validations. Also where should form validations go in general?

### CODE REVIEW

File: src/app/plugins/openstack/api/neutron.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import http from '../../../util/http'
+
+const v2Base = '/neutron/v2.0'
```
Comment (blake) [inline src/app/plugins/openstack/api/neutron.js]: This should not be hard coded. It should come from the service catalog by looking up the "network" service type, or "neutron" service name.

Comment (kevinylee1991) [reply src/app/plugins/openstack/api/neutron.js]: Oh yeah -- that should be on the road map soon. We need to add the service catalog to the simulator before we can make it non-hardcoded.

Comment (brennancheung) [reply src/app/plugins/openstack/api/neutron.js]: For now, you can make a function that returns the hard-coded URL.  Later on, we can make it get it from the catalog.  The function should probably be promised based as well.

File: src/app/plugins/openstack/components/networks/AddNetworkPage.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+const mapStateToProps = state => ({})
+
+@withRouter
+@connect(mapStateToProps)
+class AddNetworkPage extends React.Component {
+  handleSubmit = network => {
+    const { dispatch, history } = this.props
+    try {
+      dispatch(addNetwork(network))
+      history.push('/networks')
```
Comment (brennancheung) [inline src/app/plugins/openstack/components/networks/AddNetworkPage.js]: Might need to be `/ui/openstack/networks`.  Verify this works correctly.

**Comment by kevinylee1991** [comment]
squashed/merged through local cli

**Pull request closed without merging**